tests/app/views/test_search.py

```diff
@@ -31,22 +31,27 @@ def test_should_not_be_able_create_index_twice(self):
         response = self.client.put('/index-to-create')
         assert_equal(response.status_code, 400)
         assert_equal(
+            "IndexAlreadyExistsException[[index-to-create] already exists]" in
             get_json_from_response(response)["results"],
-            "IndexAlreadyExistsException[[index-to-create] already exists]")
+            True)
 
     def test_should_not_be_able_delete_index_twice(self):
         self.client.put('/index-to-create')
         self.client.delete('/index-to-create')
         response = self.client.delete('/index-to-create')
         assert_equal(response.status_code, 404)
-        assert_equal(get_json_from_response(response)["results"],
-                     "IndexMissingException[[index-to-create] missing]")
+        assert_equal(
+            "IndexMissingException[[index-to-create] missing]" in
+            get_json_from_response(response)["results"],
+            True)
 
     def test_should_return_404_if_no_index(self):
         response = self.client.get('/index-does-not-exist/status')
         assert_equal(response.status_code, 404)
-        assert_equal(get_json_from_response(response)["status"],
-                     "IndexMissingException[[index-does-not-exist] missing]")
+        assert_equal(
+            "IndexMissingException[[index-does-not-exist] missing]" in
+            get_json_from_response(response)["status"],
```

Comment:
I Changed these tests to be "in" not "==" as a change to elastic search versions breaks the tests due to whitespace, so slightly more lenient.